### PR TITLE
Crud staff

### PR DESCRIPTION
**Pull request by CherieLK**
created CRUD for Staff

### CODE REVIEW

File: src/main/java/org/launchcode/workforce/repository/StaffRepository.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.launchcode.workforce.repository;
+
+import org.launchcode.workforce.model.ClientsJobs;
```
Comment (adahathome) [inline src/main/java/org/launchcode/workforce/repository/StaffRepository.java]: You don't need this ClientsJobs import statement,

**Review by adahathome — CHANGES_REQUESTED** [review]
Looks good in general, just one edit!

**Review by tspencer0917 — APPROVED** [review]
Looks good!

File: src/main/java/org/launchcode/workforce/repository/StaffRepository.java (hunk 1 above)
Comment (CherieLK) [reply src/main/java/org/launchcode/workforce/repository/StaffRepository.java]: done - thank you

**Review by adahathome — APPROVED** [review]
Awesome, looks good

**Pull request merged**